src/lib/providers/MintProvider.tsx

```
@@ -56,6 +56,11 @@ const MintProvider: React.FC<MintProviderType> = ({ children }) => {
 	// }, []);
 
 	const mint = async (nft: NftParams, setStatus: (status: string) => void) => {
+		// @todo better validation
+		if (/[A-Z]/.test(nft.zna)) {
+			return;
```

Nit: Maybe throw an error?

---

Good call - will do